status/boost_check_library.py

```diff
@@ -83,6 +107,33 @@ def check_organization_include(self):
                 ''',
                 'org-inc-one')
     
+    def check_organization_macros(self):
+        libdir = os.path.join(self.library_dir,'include','boost',self.library_name)
+        if os.path.isdir(libdir):
+            badmacros = {}
+            for root, dirs, files in os.walk(libdir):
+                if os.path.sep + 'detail' + os.path.sep in root or root.endswith(os.path.sep + 'detail'):
+                    continue
+                for item in files:
+                    ext = os.path.splitext(item)
+                    if ext[1] == '.hpp' or ext[1] == '.ipp' or ext[1] == '.h':
+                        p = Preprocessor(self)
```

Comment:
Can you put this in a function like `getBadMacros(file)`? The nesting gets really deep

Reply:
I don't think the nesting deeper than other code in the same file.